keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -238,8 +238,8 @@ At a high level, the proposal is to modify the StatefulSet and Job controllers t
 as a pod label at pod creation time (for jobs, this would only apply to jobs in
 Indexed completion mode). The details of this are outlined in the Design Details section below.
 
-StatefulSet pod label: `statefulset.kubernetes.io/pod-index`
-Indexed Job pod label: `batch.kubernetes.io/job-completion-index` (same as existing annotation)
+- StatefulSet pod label: `apps.kubernetes.io/pod-index`
+- IndexedJob pod label: `batch.kubernetes.io/job-completion-index` (same as existing annotation)
 
 ### User Stories (Optional)
```

Comment:
Nit, in the 2nd story, there's a typo, we have:
```
Instead of creating a service which matche an entire Job
```
but should be
```
Instead of creating a service which matches an entire Job
```

Reply:
updated.